readthedocs/projects/static-src/projects/css/_content.less

```diff
@@ -213,3 +213,79 @@ select {
     padding-left: 0;
   }
 }
+
+.projects-filter {
+    &:before {
+        font-family: FontAwesome;
+        content: "\f07b";
+        padding-right: 10px;
+    }
+}
+
+.results-header {
+    dispay: block;
+    border-bottom: 2px solid rgba(0,0,0,.1);
+    padding-bottom: 20px;
+    margin-bottom: 20px;
+}
+
+.module-item.search-result {
+    border-right: 1px solid #bfbfbf;
+    border-bottom: 0;
+    padding: 15px;
+
+    &:nth-of-type(3n) {
+        border-right: 0;
+    }
+
+    hr {
+        margin: 0;
+        margin-top: 50px;
+        width: 50%;
+    }
+}
+
+.search-result-publisher {
+    text-transform: uppercase;
+    &:before {
+        font-family: FontAwesome;
+        content: "\f15c";
+        padding-right: 10px;
+        color: #0062ce;
+        font-size: 40px;
+        line-height: 1.3;
+        float: left;
+    }
+}
+
+.results-filter {
+    text-align: right;
```

Comment:
Anche in questo elemento, nell'html si dovrebbero poter usare le classi `p-0 text-right`.